config/settings/pen.yml

```diff
@@ -0,0 +1,43 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://pen.register-trainee-teachers.service.gov.uk
+
+dttp:
```

Comment:
This maybe just a copy+paste, but I believe the link to DTTP doesn't exist anymore ? Is this still required in the codebase @markhallen ?

Reply:
Correct

Reply:
I'll leave this in so it matches the other envs and it can be removed through another PR for the other envs.